docs/intro.asciidoc

```diff
@@ -10,6 +10,7 @@ Welcome to the APM .NET Agent documentation.
 
 Elastic APM automatically measures the performance of your application and tracks errors.
 For example, it records spans for database queries and transactions for incoming HTTP requests.
+See <<supported-technologies>> to learn about which frameworks and technologies are supported by Elastic APM .NET agent.
```

Comment:
@SergeyKleyman - the doc build is actually complaining about this attribute. "supported-technologies" only exists as a file name, not an attribute, so this link doesn't work.

Reply:
Thank you very much - it worked :)